bin/check-process.rb

```diff
@@ -287,7 +300,7 @@ def run
 
     if config[:metric]
       # #YELLOW
-      count = procs.map { |p| p[config[:metric]].to_i }.reduce { |a, b| a + b } # rubocop:disable SingleLineBlockParams
+      count = procs.map { |p| p[config[:metric]].to_i }.reduce { |a, e| a + e }
```

Comment:
This comes from [Style/SingleLineBlockParams](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/SingleLineBlockParams). [Here are the tests](https://github.com/bbatsov/rubocop/blob/master/spec/rubocop/cop/style/single_line_block_params_spec.rb) rubocop runs for an example of what led to the message:
```
Name `reduce` block params `|a, e|`.
```

I'm perfectly happy to put things back how I found them, but I guessed that a previous committer may not have fully understood the SingleLineBlockParams cop and could have avoided the `rubocop:disable`. Switching `b` to `e` makes everything happy.

Might have had something to do with my editor vs. running rubocop from the CLI, but I ran `bundle exec rubocop` and needed to make this change for it to pass locally, though Atom was perfectly happy before I made the change.

Let me know how to proceed.